src/main/java/htmlpublisher/HtmlPublisher.java

```diff
@@ -47,6 +47,7 @@
 
 import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
 import hudson.util.DirScanner;
+import io.jenkins.cli.shaded.org.apache.commons.lang.StringUtils;
```

Comment:
could we use the normal commons-lang package rather than the shaded one from core